src/mlpack/bindings/R/CMakeLists.txt

```diff
@@ -203,6 +203,11 @@ if (BUILD_R_BINDINGS)
        DESTINATION
        "${CMAKE_CURRENT_BINARY_DIR}/mlpack/")
 
+  file(COPY
+       "${CMAKE_CURRENT_SOURCE_DIR}/mlpack/configure"
+       DESTINATION
+       "${CMAKE_CURRENT_BINARY_DIR}/mlpack/")
+
```

Comment:
Why do we need to copy it in?  Can we not keep it there?

Reply:
Ah, this just copies the configure script from the source directory to the build directory.  For CMake that's a necessary step, since it does out-of-source builds. :+1: